packages/react-cards/src/components/Card/CardSection/CardSection.types.ts

```diff
@@ -0,0 +1,74 @@
+import { IComponent, IComponentStyles, ISlotProp, IStyleableComponentProps } from '@uifabric/foundation';
+import { IBaseProps } from '@uifabric/utilities';
+import { IStackProps, IStackSlots, IStackTokens } from 'office-ui-fabric-react/lib/Stack';
+
+/**
+ * {@docCategory CardSection}
+ */
+export type ICardSectionComponent = IComponent<ICardSectionProps, ICardSectionTokens, ICardSectionStyles, ICardSectionViewProps>;
+
+// These types are redundant with ICardSectionComponent but are needed until TS function return widening issue is resolved:
+// https://github.com/Microsoft/TypeScript/issues/241
+// For now, these helper types can be used to provide return type safety when specifying tokens and styles functions.
+
+/**
+ * {@docCategory CardSection}
+ */
+export type ICardSectionTokenReturnType = ReturnType<Extract<ICardSectionComponent['tokens'], Function>>;
+
+/**
+ * {@docCategory CardSection}
+ */
+export type ICardSectionStylesReturnType = ReturnType<Extract<ICardSectionComponent['styles'], Function>>;
+
+/**
+ * {@docCategory CardSection}
+ */
+export type ICardSectionSlot = ISlotProp<ICardSectionProps>;
+
+/**
+ * {@docCategory CardSection}
+ */
+export interface ICardSectionSlots extends IStackSlots {}
+
+/**
+ * {@docCategory CardSection}
+ */
+export interface ICardSection {}
+
+/**
+ * {@docCategory CardSection}
+ */
+export interface ICardSectionProps
+  extends ICardSectionSlots,
+    Pick<
+      IStackProps,
+      'as' | 'horizontal' | 'reversed' | 'horizontalAlign' | 'verticalAlign' | 'verticalFill' | 'disableShrink' | 'grow' | 'wrap'
+    >,
```

Comment:
Would this be easier if it just extends all of IStackProps? I'm thinking that would be the better default, unless there are props we specifically want to exclude.

Reply:
Just extending `IStackProps` won't work since things like `styles` and `tokens` would be defined twice in the interface.